### DISCUSSION TITLE
How to change the user agent when loading sites in webview?

### DISCUSSION THREAD

**Discussion by kishaningithub** [tauri-apps/tauri#3519] [Q&A] [answered]
When i load sites via webview, some sites say the browser version is not supported.

Note - I am using latest version of mac os x with every update installed

**Google chat**
<img width="600" alt="Screenshot 2022-02-20 at 10 12 29 AM" src="https://user-images.githubusercontent.com/763760/154828696-d35cd402-dbc1-4a93-bfcb-5436584b70f1.png">

**Whatsapp**
<img width="600" alt="Screenshot 2022-02-20 at 10 13 11 AM" src="https://user-images.githubusercontent.com/763760/154828707-be7ee315-4005-4bf5-8d63-fce8deb42775.png">

My current code is a simple anchor link using the create-react-app starter

```html
<a
  className="App-link"
  href="https://web.whatsapp.com"
  rel="noopener noreferrer"
>
  Whatsapp
</a>
<a
  className="App-link"
  href="https://chat.google.com"
  rel="noopener noreferrer"
>
  Google chat
</a>
```

**Comment by FabianLars** [comment, marked as answer]
There was a config introduced in v1.2 for that: https://tauri.app/v1/api/config#windowconfig.useragent
